cfn-templates/cid-cfn.yml

```diff
@@ -61,6 +61,8 @@ Metadata:
         default: "Deploy KPI Dashboard"
       OptimizationDataCollectionBucketPath:
         default: "Path to Optimization Data Collection S3 bucket"
+      MonthsLookback:
```

Comment:
This only affects summary view, right? Not sure what what would be a right name here. 

* SummaryViewDefaultMonthLimit ?
* DefaultMonthLimit ?

Also need to make it clear that updating of this parameter wont update the views/datasets/dashboards